tests/Http/ReviewsTest.php

```diff
@@ -70,9 +69,7 @@ public function testUnauthenticatedUserCantReviewPosts()
     public function testUpdatedPostAndSignupWithReview()
```

Comment:
do we need this test anymore?

Reply:
Hmm, not sure! I _thiiiink_ we still want these until we can move to a "push" model for getting updates to the data team but then we can probs remove?